recipe/meta.yaml

```diff
@@ -1,24 +1,35 @@
-{% set version = "1.10" %}
-{% set sha256 = "bcbe8ada8c435a253b1fa438f505dd311832a88f52316fd7d30342b106a4242c" %}
-{% set runc_version = "0.1.0" %}
-{% set runc_sha256 = "78ae42905c0f26fcfa9353e48e1a001b5b5c3c94b33e2ceacc6bc84c23c61a88" %}
+{% set goname = "github.com/tianon/gosu" %}
+{% set version = "1.11" %}
+
+{% set name = goname.split('/')[-1] %}
+{% set pkg_src = 'src/'+goname %}
+
+# The runc version much match the upstream Dockerfile
+# https://github.com/tianon/gosu/blob/master/Dockerfile
+{% set runc_goname = "github.com/opencontainers/runc" %}
+{% set runc_version = "1.0.0-rc5" %}
+{% set runc_pkg_src = 'src/'+runc_goname %}
 
 package:
-  name: gosu
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  - folder: src/github.com/tianon/gosu
-    url: https://github.com/tianon/gosu/archive/{{ version }}.tar.gz
-    sha256: {{ sha256 }}
+  - folder: {{ pkg_src }}
+    url: https://{{ goname }}/archive/{{ version }}.tar.gz
+    sha256: d03208fd7ef983e4a606f861a5efcdc5a1980952a91febba3f67057800da6701
 
-  - folder: src/github.com/opencontainers/runc
-    url: https://github.com/opencontainers/runc/archive/v{{ runc_version }}.tar.gz
-    sha256: {{ runc_sha256 }}
+  - folder: {{ runc_pkg_src }}
+    url: https://{{ runc_goname }}/archive/v{{ runc_version }}.tar.gz
+    sha256: 770e180d1cca1f3c1a67512ebf9720caafe97af4fc2fe12fa43a3cf7bfa77522
 
 build:
-  number: 1001
+  number: 1000
   skip: true  # [not linux]
+  script:
+    - pushd {{ pkg_src }}
+    - go install -v -ldflags '-d -s -w'
```

Comment:
Should we cleanup the Go package cache? Does that wind up in the `PREFIX`?

Reply:
No need to clean-up the cache with our current CI setup. The go package cache is based on the GOPATH variable which is set to $SRC_DIR. 

I built the package locally and here is list of files to verify:
```
" tar.vim version v29
" Browsing tarfile /opt/conda/conda-bld/linux-64/gosu-1.11-h375a9b1_1000.tar.bz2
" Select a file with cursor and press ENTER

info/files
info/hash_input.json
info/paths.json
info/index.json
info/about.json
info/LICENSE.txt
info/git
info/test/run_test.sh
info/recipe/conda_build_config.yaml
info/recipe/meta.yaml
bin/gosu
info/recipe/meta.yaml.template
```

Reply:
Ah ok. Thanks for the reminder. Looks good.

Reply:
👍